frontend/13f-app/src/components/pages/datapage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import FilingData from '../filingdata'
```

Comment:
```suggestion
import FilingData from '../FilingData'
```
Change the filename accordingly.